.github/workflows/ci-it.yml

```diff
@@ -196,7 +196,7 @@ jobs:
           GOLANG_PROTOBUF_REGISTRATION_CONFLICT: ignore
         run: |
           mkdir -p ./coverage
-          go test -tags=musl -work -timeout=10s -failfast -race -cover -coverprofile=./coverage/${{ steps.cov-report-name.outputs.path }} -covermode=atomic -gcflags='-l -N' ${{ matrix.package-paths }}
```

Comment:
10s is enough

Reply:
instead of dialer server health check with timeout, ensure that cluster-Agent can be connected successfully at one time